libraries/package_deps.rb

```diff
@@ -8,27 +8,27 @@ def openssl_prefix
     module PackageDeps
       def cruby_package_deps
         case node['platform_family']
-        when 'rhel', 'fedora', 'amazon'
-          %w( gcc bzip2 openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel gdbm-devel ncurses-devel make )
+        when 'rhel', 'fedora', 'amazon', 'oracle'
```

Comment:
Looks like Oracle isn't a valid platform, someone should tell Oracle 😄 

If you can remove that from the platform matrix that'd be great.

Reply:
Oracle doesn't really need to be there, but it does work as a platform on my side. Strange that it doesn't show up as valid. Will fix.

Reply:
Oracle removed from libraries/package_deps.rb.